apps/idos-enclave/src/lib/enclave.js

```diff
@@ -20,6 +21,10 @@ export class Enclave {
     this.#listenToRequests();
   }
 
+  get isAuthorizedOrigin() {
+    return this.authorizedOrigins.includes(this.parentOrigin);
+  }
```

Comment:
Style: unclear if this is small enough work to make this an attribute. But, meh, 🤷, up to you.